Documentation/Internationalisation/Crowdin.md

```diff
@@ -0,0 +1,16 @@
+## Translations via Crowdin
+
+The Issue Tracker proudly welcomes translations contributed via [Crowdin](https://crowdin.com/).  The project can be found at [https://crowdin.com/project/joomla-official-sites](https://crowdin.com/project/joomla-official-sites).
+
+#### Crowdin Guidelines
+
+Log on to [Crowdin's Help Center] (https://support.crowdin.com/) to find useful information and resources for Translators as well as for Developers. 
+You can't find the answer to your question? Please contact [Marc-Antoine Thevenet] (https://crowdin.com/mail/compose/MAT978).
+
+#### Adding languages
+
+If your language is not yet available on Crowdin, please contact [Marc-Antoine Thevenet] (https://crowdin.com/mail/compose/MAT978) or [Michael Babker] (https://crowdin.com/mail/compose/mbabker) on Crowdin so that they can set it up.
+And if your language is already 100% translated, don't hesitate to improve or comment the translations or you can simply vote for your favorite translation.
+
+#### Crowdin: Other Official Joomla! Projects
+Other official resources are available for translation on Crowdin. Just have a look [here] (https://crowdin.com/projects/Joomla) and help us with the internalization of Joomla.
```

Comment:
favorite =>favourite please


On 10 July 2016 at 17:37, Brian Teeman <brian@teeman.net> wrote:

> internalization ???
>
>
> On 10 July 2016 at 17:34, Nikolai Plath <notifications@github.com> wrote:
>
>> In Documentation/Internationalisation/Crowdin.md
>> <https://github.com/joomla/jissues/pull/827#discussion_r70184707>:
>>
>> > +## Translations via Crowdin
>> > +
>> > +The Issue Tracker proudly welcomes translations contributed via [Crowdin](https://crowdin.com/).  The project can be found at [https://crowdin.com/project/joomla-official-sites](https://crowdin.com/project/joomla-official-sites).
>> > +
>> > +#### Crowdin Guidelines
>> > +
>> > +Log on to [Crowdin's Help Center] (https://support.crowdin.com/) to find useful information and resources for Translators as well as for Developers.
>> > +You can't find the answer to your question? Please contact [Marc-Antoine Thevenet] (https://crowdin.com/mail/compose/MAT978).
>> > +
>> > +#### Adding languages
>> > +
>> > +If your language is not yet available on Crowdin, please contact [Marc-Antoine Thevenet] (https://crowdin.com/mail/compose/MAT978) or [Michael Babker] (https://crowdin.com/mail/compose/mbabker) on Crowdin so that they can set it up.
>> > +And if your language is already 100% translated, don't hesitate to improve or comment the translations or you can simply vote for your favorite translation.
>> > +
>> > +#### Crowdin: Other Official Joomla! Projects
>> > +Other official resources are available for translation on Crowdin. Just have a look [here] (https://crowdin.com/projects/Joomla) and help us with the internalization of Joomla.
>>
>> Well, the only glitch I was able to spot with my Denglish eyes is that
>> Joomla! should probably wear its "*!*" here...
>> ! 😜
>>
>> Otherwise this looks great.
>>
>> —
>> You are receiving this because you are subscribed to this thread.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/joomla/jissues/pull/827/files/3af1d8eedf8d1c0390654c98b2a19046f040b61d#r70184707>,
>> or mute the thread
>> <https://github.com/notifications/unsubscribe/ABPH8VnT2LUyrRdbceVUz6eo54lcxKGIks5qUR74gaJpZM4JI3Fb>
>> .
>>
>
>
>
> --
> Brian Teeman
> Co-founder Joomla! and OpenSourceMatters Inc.
> http://brian.teeman.net/
>



-- 
Brian Teeman
Co-founder Joomla! and OpenSourceMatters Inc.
http://brian.teeman.net/